src/model/transferlog.js

```diff
@@ -9,25 +9,48 @@ const AlreadyFulfilledError = errors.AlreadyFulfilledError
 const MissingFulfillmentError = errors.MissingFulfillmentError
 const debug = require('debug')('ilp-plugin-virtual:store')
 
+const TRANSFER_PREFIX = 'transfer_'
+
 module.exports = class TransferLog {
 
   constructor (opts) {
     this._store = opts.store
+    this._storeCache = {}
+  }
+
+  _get (transferId) {
+    debug('_get', transferId)
+    return this._storeCache[transferId]
+  }
+
+  _put (id, packaged) {
```

Comment:
I find this file kind of confusing to follow because there are 3 functions that sound like they'd do almost the same thing but don't: `_put`, `_write`, `_storeTransfer`, `_storePackaged`. Could you put some comments to explain what they're doing and/or use more descriptive function names?